src/app/contexts/ToggleContext/index.test.jsx

```diff
@@ -107,6 +140,7 @@ describe('ToggleContext with feature toggles', () => {
       remoteAdToggleValue,
       service,
       remoteToggleExpectation,
+      remoteAdvertiseCombinedValue,
       renderExpectation,
     }) => {
       describe(`given service is ${service}`, () => {
```

Comment:
I think we should also update the describe text on line 147, to include some context about `remoteAdvertiseCombinedValue`